### ISSUE TITLE
Add international space station

### ISSUE THREAD

**Issue by TysonMN** [sky-map-team/stardroid#318]
Has anyone considered adding the international space station to the list of things visable through this app?

I have never seen it before (mostly because I have never looked for it), but I recently learned that it is about the brightness of Venus (so very easy to see).  I would love if this app could help me locate the international space station in the night sky.

**Comment by hildogjr** [comment]
I would say "show/track artificial satellites (Internet connection needed)". If possible show more than just the ISS.

**Comment by ildar** [comment]
If implemented,  https://github.com/sky-map-team/stardroid/issues/327 can somewhat help (a little)

**Comment by adanhamidu** [comment]
@bender2k14 I guess you could use "[ISS Detector](https://play.google.com/store/apps/details?id=com.runar.issdetector) " app until ISS tracking becomes available in this app.

**Comment by bender2k14** [comment]
Yes, that is what I have been doing.  It works great.

**Comment by jaydeetay** [comment]
Yes - this would be a great feature. I've been delaying it because the whole UI and datamodel needs some reworking. However, if all we want is a dot on the map (just like the Messier objects, for example) then we don't need to wait for that.  I've created a new branch and I'll start to do some research. IIRC we even had some code for this from ten years ago that someone wrote to make a start.

**Comment by GuiSousa135** [comment]
#478

**Comment by jaydeetay** [comment]
I did a bit of research and for objects in such a low orbit we'd need to update the orbital parameters every few days to be even within a few degrees.  So this isn't something that can be "baked in" to the app - it requires code to pull the latest parameters over the internet.  Very do-able, but not something we can hack in.  I'd hoped that maybe updating them with a monthly release cadence would be enough, but it won't be.

**Comment by jaydeetay** [comment]
Duplicate of #478. Same ask — adding the ISS to the map. Consolidating discussion there — closing this one.
